### ISSUE TITLE
[1.17 Bug Bash] [Win10] Cannot keyboard navigate through nav view items

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/terminal#14691]
Tested on VB Release N.

commit SHA: 3034d01c5bdcbfbf17493b6bed0e0db301ed4a0d

**Comment by zadjii-msft** [comment]
This is certainly a regression in #14630 so I should just fix this there

**Comment by zadjii-msft** [comment]
I'm just gonna roll this in with the parent PR.
